frontend/src/__tests__/cypress/cypress/pages/enabled.ts

```diff
@@ -0,0 +1,52 @@
+class EnabledPage {
+  visit() {
+    cy.visitWithLogin('/');
+    this.wait();
+  }
+
+  private wait() {
+    cy.findByTestId('enabled-application').should('be.visible');
+    cy.testA11y();
+  }
+
+  findPageTitle() {
+    return cy.findByTestId('app-page-title');
+  }
+
+  shouldHavePageDescription() {
+    return cy
+      .findByTestId('app-page-description')
+      .contains(
+        'Launch your enabled applications, view documentation, or get started with quick start instructions and tasks.',
+      );
+  }
+
+  findEnabledCard() {
+    return cy.findByTestId('jupyter');
+  }
```

Comment:
There can be multiple cards on the page. Do not hard code dynamic values into the page objects. Those should be passed in as parameters to the function.

I've created an enhancement to `findByTestId` to better support this use case. See https://github.com/opendatahub-io/odh-dashboard/pull/2579

```suggestion
  findCard(id: string) {
    return cy.findByTestId(['card', id]);
  }
```

Reply:
It's also possible to do:
``cy.findByTestId(`card ${id}`)``